common/views/pages/_app.js

```diff
@@ -153,9 +157,18 @@ export default class WecoApp extends App {
     if (isClient && !isPreview) {
       isPreview = props.isPreview;
     }
+
     super(props);
   }
 
+  state: State = {
+    togglesContext: toggles,
+  };
+
+  updateToggles = (newToggles: Object) => {
```

Comment:
We're moving the functionality of the toggles into even more places here.
The component is setting the cookie, here it's updating the context, and it's being read by some middleware.

I wonder if we create a toggles context that can do all of this in one place, simplifying the component reading and writing to this.

This works - and happy to move this conversation into another PR.

Reply:
👍